CallAutomation_OutboundCalling/README.md

```diff
@@ -0,0 +1,45 @@
+|page_type| languages                             |products
+|---|---------------------------------------|---|
+|sample| <table><tr><td>Java</tr></td></table> |<table><tr><td>azure</td><td>azure-communication-services</td></tr></table>|
+
+# Call Automation - Quick Start Sample
+
+This sample application shows how the Azure Communication Services  - Call Automation SDK can be used to build IVR related solutions. 
+It makes an outbound call to a phone number, performs DTMF recognition, plays a different audio message based on the key pressed by the callee and hangs-up the call. 
+This sample application configured for accepting tone 1 (tone1), 2 (tone2) , If the callee pressed any other key than expected, the call will be disconnected.
+This sample application is also capable of making multiple concurrent outbound calls. The application is a web-based application built on Java's Spring framework.
+
+# Design
+
+![design](./static/OutboundCallDesign.png)
+
+## Prerequisites
+
+- An Azure account with an active subscription. [Create an account for free](https://azure.microsoft.com/free/?WT.mc_id=A261C142F). 
+- A deployed Communication Services resource. [Create a Communication Services resource](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource).
+- A [phone number](https://learn.microsoft.com/en-us/azure/communication-services/quickstarts/telephony/get-phone-number) in your Azure Communication Services resource that can make outbound calls. NB: phone numbers are not available in free subscriptions.
+- Create an Azure account with an active subscription. For details, see [Create an account for free](https://azure.microsoft.com/free/)
+- [Java Development Kit (JDK) version 11 or above](https://docs.microsoft.com/azure/developer/java/fundamentals/java-jdk-install)
+- [Apache Maven](https://maven.apache.org/download.cgi)
+- Download and install [Ngrok](https://www.ngrok.com/download). As the sample is run locally, Ngrok will enable the receiving of all the events.
```

Comment:
Can we use Azure devtunnel CLI instead of ngrok to keep in sync with other quickstarts.

Reply:
I don't think we should for Java, not all Java devs will want to use Visual Studio for this purpose only. Ngrok is more commonly used from my point of view.

Reply:
I don't think we should for Java, not all Java devs will want to use Visual Studio for this purpose only. Ngrok is more commonly used from my point of view.

Reply:
forgot this

Reply:
fixed, please recheck. Thanks